modules/icgc-argo-workflows/prep/metrics/resources/usr/bin/dnaaln.py

```diff
@@ -47,7 +47,7 @@
      'total_pf_reads': 'sequences',
      'average_base_quality': 'average_quality',
      'average_read_length': 'average_length',
-     'reads_duplicated_percent': 'reads_duplicated_percent',
+     'pct_reads_duplicated': 'reads_duplicated_percent',
```

Comment:
We handle `percent` as `pct` for standardization. This one was missed. Now correct it to the standard format